### PR TITLE
feat(component): added SearchSelect component

### PR DESCRIPTION
**Pull request by noahschechter**

### CODE REVIEW

File: src/components/SearchSelect/SearchSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  idRef,
+  ignoreCasing = false
+}) => {
+  const [searchValue, setSearchValue] = useState('');
+  const filteredOptions = useMemo(() => {
+    if (!searchValue.length) {
+      return options;
+    }
+
+    return options.filter(o =>
```
Comment (matanmoda) [inline src/components/SearchSelect/SearchSelect.tsx:33]: how'bout option? :)

**Review by matanmoda — APPROVED** [review]
Approving cause it looks good.. feel free to change that option comment (or not)

File: src/components/SearchSelect/SearchSelect.tsx (hunk 1 above)
Comment (noahschechter) [reply src/components/SearchSelect/SearchSelect.tsx:33]: will do

**Review by hevele-moda — COMMENTED** [review]
LGTM, just thinking on the UX here, wouldn't it be better if the search input was in the dropdown that opens when you click on the select?
And it can maybe be part of the `Select` component like `<Select options={...} searchable />`

**Comment by noahschechter** [comment]
> LGTM, just thinking on the UX here, wouldn't it be better if the search input was in the dropdown that opens when you click on the select?
> And it can maybe be part of the `Select` component like `<Select options={...} searchable />`

Yeah actually that looks way better. I just googled and found this https://www.npmjs.com/package/react-select-search which is the same way. I can try to make a modification

**Pull request closed without merging**